proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml

```diff
@@ -33,15 +33,15 @@
                 </hint>
             </perform>
         </rule>
-        <rule id="acme-servlet-annotations_002">
+        <rule id="proprietary-servlet-annotations_002">
```

Comment:
@lincolnthree : Are these related rules supposed to be 1000 apart? I thought it would be -01000, -01001, -01002 since they are related rules. Then the next unrelated batch would be -02000. If we space them all 1000 apart, what's the point of the '000' position?